pkg/tsdb/azuremonitor/azure-log-analytics-datasource.go

```diff
@@ -165,10 +155,9 @@ func (e *AzureLogAnalyticsDatasource) executeQuery(ctx context.Context, query *A
 
 	span, ctx := opentracing.StartSpanFromContext(ctx, "azure log analytics query")
 	span.SetTag("target", query.Target)
-	span.SetTag("from", timeRange.From)
-	span.SetTag("until", timeRange.To)
-	span.SetTag("datasource_id", e.dsInfo.Id)
-	span.SetTag("org_id", e.dsInfo.OrgId)
```

Comment:
Couple of comments of these tags:
 - Before, `from` and `until` were set in ms, now it's nano seconds (just because there is a convenient method for that). Is this change okay?
  - The `OrgId` info is no longer available (at least I could not find it nor in the setings or the query). Is it okay to remove it?

Reply:
WDYT @marefr?

Reply:
No clue how to do this, but I guess it should be possible to get org id. Do you know @marefr?

Reply:
It's available on the [PluginContext](https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go@v0.103.0/backend#PluginContext)

Reply:
To get millisecond resolution you can do `query.TimeRange.From.UTC().UnixNano() /  int64(time.Millisecond)`. Not sure you actually need the additional `.UTC()`?